### ISSUE TITLE
Add gRPC function to visualize TextFSM result dictionary as Freeplane (mindmap) nodes

### ISSUE THREAD

**Issue by metacoma** [metacoma/mindwm#39]
The TextFSM produces a dictionary that contains the objects and properties parsers from processing the input and output (IO context) command in a tmux pane. 

As a result, we need to implement a function that can visualize the TextFSM dictionary as Freeplane nodes, which will help us better understand the data. 

The function should draw the structures that output, but jsonify the TextFSM result dictionary.

**Comment by metacoma** [comment]
The following table is the result of the textfsm parsing of the 'ip a show' command."

textFSM rule https://github.com/networktocode/ntc-templates/blob/master/ntc_templates/templates/linux_ip_address_show.textfsm

python code: 
```
   if (patterns["{{ ctx.input }}"].match(input_data)):
          textfsm_template="""{{ ctx.textfsm }}"""
          re_table = textfsm.TextFSM(io.StringIO(textfsm_template))
          header = re_table.header
          try:
              result = re_table.ParseText(output_data)
              print(tabulate(result, headers=header))
          except textfsm.parser.TextFSMError:
              continue
```

``` ID  INTERFACE    FLAGS                                      MTU  QDISC     STATE    GROUP    QLEN    MASTER    TYPE      ADDRESS            BROADCAST          IP_ADDRESSES        IP_MASKS    IPV6_ADDRESSES                 IPV6_MASKS                                                                                                                                                  
----  -----------  ---------------------------------------  -----  --------  -------  -------  ------  --------  --------  -----------------  -----------------  ------------------  ----------  -----------------------------  ------------                                                                                                                                                
   1  lo           LOOPBACK,UP,LOWER_UP                     65536  noqueue   UNKNOWN  default  1000              loopback  00:00:00:00:00:00  00:00:00:00:00:00  ['127.0.0.1']       ['8']       ['::1']                        ['128']                                                                                                                                                     
   2  ens160       BROADCAST,MULTICAST,UP,LOWER_UP           1500  fq_codel  UP       default  1000              ether     00:50:56:9f:32:ec  ff:ff:ff:ff:ff:ff  ['217.61.104.247']  ['24']      ['fe80::250:56ff:fe9f:32ec']   ['64']                                                                                                                                                      
   3  ens192       BROADCAST,MULTICAST                       1500  noop      DOWN     default  1000              ether     00:50:56:9f:0a:e3  ff:ff:ff:ff:ff:ff  []                  []          []                             []                                                                                                                                                          
   4  ens224       BROADCAST,MULTICAST                       1500  noop      DOWN     default  1000              ether     00:50:56:9f:f6:f7  ff:ff:ff:ff:ff:ff  []                  []          []                             []                                                                                                                                                          
   5  tun0         POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP   1500  fq_codel  UNKNOWN  default  100               none                                            ['10.8.0.1']        ['24']      ['fe80::e53e:9943:b064:f23a']  ['64']
   6  wg0          POINTOPOINT,NOARP,UP,LOWER_UP             1420  noqueue   UNKNOWN  default  1000              none                                            ['10.0.88.1']       ['24']      []                             []
   7  docker0      NO-CARRIER,BROADCAST,MULTICAST,UP         1500  noqueue   DOWN     default                    ether     02:42:7a:15:37:7e  ff:ff:ff:ff:ff:ff  ['172.17.0.1']      ['16']      ['fe80::42:7aff:fe15:377e']    ['64']
```

**Comment by metacoma** [comment]
and here JSON representation
```json
{"header": ["ID", "INTERFACE", "FLAGS", "MTU", "QDISC", "STATE", "GROUP", "QLEN", "MASTER", "TYPE", "ADDRESS", "BROADCAST", "IP_ADDRESSES", "IP_MASKS", "IPV6_ADDRESSES", "IPV6_MASKS"], "result": [["1", "lo", "LOOPBACK,UP,LOWER_UP", "65536", "noqueue", "UNKNOWN", "default", "1000", "", "loopback", "00:00:00:00:00:00", "00:00:00:00:00:00", ["127.0.0.1"], ["8"], ["::1"], ["128"]], ["2", "ens160", "BROADCAST,MULTICAST,UP,LOWER_UP", "1500", "fq_codel", "UP", "default", "1000", "", "ether", "00:50:56:9f:32:ec", "ff:ff:ff:ff:ff:ff", ["217.61.104.247"], ["24"], ["fe80::250:56ff:fe9f:32ec"], ["64"]], ["3", "ens192", "BROADCAST,MULTICAST", "1500", "noop", "DOWN", "default", "1000", "", "ether", "00:50:56:9f:0a:e3", "ff:ff:ff:ff:ff:ff", [], [], [], []], ["4", "ens224", "BROADCAST,MULTICAST", "1500", "noop", "DOWN", "default", "1000", "", "ether", "00:50:56:9f:f6:f7", "ff:ff:ff:ff:ff:ff", [], [], [], []], ["5", "tun0", "POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP", "1500", "fq_codel", "UNKNOWN", "default", "100", "", "none", "", "", ["10.8.0.1"], ["24"], ["fe80::e53e:9943:b064:f23a"], ["64"]], ["6", "wg0", "POINTOPOINT,NOARP,UP,LOWER_UP", "1420", "noqueue", "UNKNOWN", "default", "1000", "", "none", "", "", ["10.0.88.1"], ["24"], [], []], ["7", "docker0", "NO-CARRIER,BROADCAST,MULTICAST,UP", "1500", "noqueue", "DOWN", "default", "", "", "ether", "02:42:7a:15:37:7e", "ff:ff:ff:ff:ff:ff", ["172.17.0.1"], ["16"], ["fe80::42:7aff:fe15:377e"], ["64"]]]}```

**Comment by metacoma** [comment]
Closed by #44 
![output](https://user-images.githubusercontent.com/5146707/230055082-d49cf79c-4337-4409-bbd4-a24e4f8d5637.gif)
